ooiservices/app/config.yml

```diff
@@ -55,7 +56,9 @@ COMMON: &common
     ALFRESCO_REPO_ID:   'c161bc66-4f7e-4a4f-b5f2-aac9fbf1d3cd'
     ALFRESCO_USERNAME:  'admin'
     ALFRESCO_PASSWORD:  'admin'
-
+    # cilogon
+    CILOGON_ID: 'myproxy:oa4mp,2012:/client_id/84e7951a567243a62ae8bc3e13c12c8'
```

Comment:
I think we should obfuscate these parameters (security).  Please make them generic 'xxx...'